tiledb/CMakeLists.txt

```diff
@@ -183,8 +183,63 @@ set(TILEDB_CORE_SOURCES
 if (TILEDB_SERIALIZATION)
   list(APPEND TILEDB_CORE_SOURCES
     ${TILEDB_CORE_INCLUDE_DIR}/tiledb/sm/rest/curl.cc
-    ${TILEDB_CORE_INCLUDE_DIR}/tiledb/sm/serialization/tiledb-rest.capnp.c++
   )
+
+  if(WIN32)
+    set(TILEDB_CAPNPROTOEXEPATH "${TILEDB_EP_BASE}/install/bin/capnp.exe")
+  else()
+    set(TILEDB_CAPNPROTOEXEPATH "${TILEDB_EP_BASE}/install/bin/capnp")
+  endif()
+
+  if(WIN32)
+    set(TILEDB_TMPCMD_PATH "${TILEDB_EP_BASE}/install/bin;%PATH%")
+  else()
+    set(TILEDB_TMPCMD_PATH "${TILEDB_EP_BASE}/install/bin:$PATH")
+  endif()
+
+  set(TILEDB_CAPNPROTOGENCMDARGS " compile -I ${TILEDB_EP_BASE}/install/include -oc++ tiledb-rest.capnp")
+  set(TILEDB_CAPNPROTOGENCMD "${TILEDB_CAPNPROTOEXEPATH} ${TILEDB_CAPNPROTOGENCMDARGS}")
+  if(TILEDB_CAPNP_EP_BUILT)
+    set(tiledb_runcapnproto_depends ep_capnp)
+  else()
+#####TBD: What must it be when superbuild did not build?
+    set(tiledb_runcapnproto_depends "?") 
```

Comment:
I don't know if this is valid. It should probably be an explicit error for now rather than something that might precariously break: `message(FATAL_ERROR "<msg>")`

Reply:
that and section above it is now removed
(as it is was left from my unsuccessful efforts to directly execute capnp in custom command/target COMMANDs, )

Reply:
thanks